templates/login.html

```diff
@@ -11,8 +11,10 @@
     <button class="mdl-button mdl-js-button mdl-button--colored" type="submit">
       Login
     </button>
+	<a href="/create_user.html">
```

Comment:
It doesn't look clean to me to wrap a \<button\> in an \<a\>. I *think* it should work to simply change the \<button\> to an \<a\> (with the new href and the old class). [This page](http://webdesign.tutsplus.com/tutorials/learning-material-design-lite-buttons--cms-24593) is the best I could find just now, but there might be a better resource somewhere (though I fear this goal might simply contradict the material design spec. In that case *shrug*, for now, probably).